misk-crypto/src/main/kotlin/misk/crypto/CryptoModule.kt

```diff
@@ -44,6 +47,12 @@ class CryptoModule(
               .toProvider(AeadEnvelopeProvider(key, config.kms_uri))
               .`in`(Singleton::class.java)
         }
+        KeyType.DAEAD -> {
```

Comment:
I'm really happy with now easy it seems to add another primitive to `misk-crypto`